app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,8 @@
+<h3>Add Comment</h3>
```

Comment:
- [ ] This form should be in the `new.html.erb` file, kindly consider creating the file (new.html.erb) and referring to this file, or renaming this file to `new.html.erb`

Reply:
I'd prefer to now render a new form for just adding like to a post. This would be overwhelmed in sense on UX and UI

Reply:
@petushka1, it's a requirement to have the new.html.erb, that's why I requested this change